cmd/virtlet_log/virtlet_log.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2017 Mirantis
```

Comment:
@miha-plesko May I suggest adding XLAB to the copyright here and in other new files? Make sure it says just 2017 (in some cases below I saw 2016-2017).

@pigmej is this acceptable to you?